src/classes/CRLP_RollupProcessor.cls

```diff
@@ -523,6 +522,11 @@ public inherited sharing class CRLP_RollupProcessor {
         if (mode == CRLP_RollupProcessingOptions.BatchJobMode.SkewMode && this.useForceSkewModeField && !isForceSkewModeTrue) {
             updatedRecord.put(this.qualifiedSkewField, true);
             needsUpdate = true;
+
+        // Reset any flags erroneously set for NonSkewMode records
+        } else if (mode == CRLP_RollupProcessingOptions.BatchJobMode.NonSkewMode && this.useForceSkewModeField && isForceSkewModeTrue) {
```

Comment:
Will Account record with the ForceSkewMode flag set to true still be included in the NonSkew batch job for Account Hard Credit? If so, then this logic will run and reset the flag. However, if that logic is using the flag to build the queryLocator on the batch job then those will continue to run in the SkewMode job and not be picked up here.

Reply:
Account records with the ForceSkewMode flag will not be picked up to run in skew mode by the nightly run on the basis of the ForceSkewMode flag.  If they qualify on the basis of threshold settings, they will run in skew mode and don't need the flag reset.  Account records which should qualify for nonskew batch runs on the basis of the threshold settings will be picked up to run in nonskew mode by the nightly jobs and the flag will be reset.  The ForceSkewMode flag will become the basis for selection in our nightly jobs in the next work item.  Sound right?

Reply:
Yep. Sounds perfect. Thank you for confirming.